src/Organic/Organic.php

```diff
@@ -23,6 +23,7 @@ class Organic {
     public $version = \Organic\ORGANIC_PLUGIN_VERSION;
 
     const DEFAULT_PLATFORM_URL = 'https://app.organic.ly';
+    const DEFAULT_REST_API_URL = 'https://api.organic.ly';
```

Comment:
Is there any reason we need to specify REST?

Reply:
Technically no, but the environment variable `ORGANIC_API_URL` currently points to the graphql url I believe, so I named it that to avoid confusion.

Reply:
I see. If we're phasing out the REST URL anyway, maybe we can just use the `ORGANIC_API_URL` as is. But I don't have strong opinions here.

Reply:
We would still need REST API for things like [ads.txt and sellers.json](https://github.com/orgnc/platform/blob/master/backend/ads/api/rest.py#L41-L79)

But in this case, we should use `https://api.organiccdn.io/` instead
https://organicly.atlassian.net/wiki/spaces/NTE/pages/3080361/Fastly+setup+for+organiccdn.io+and+app.organic.ly